packages/node_modules/pouchdb/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "6.1.0-prerelease",
   "description": "PouchDB is a pocket-sized database",
   "main": "./lib/index.js",
-  "jsnext:main": "./lib/index-es.js",
+  "jsnext:main": "./lib/index.es.js",
+  "module": "./lib/index.es.js",
```

Comment:
I'm not comfortable with adding "module", since we're not doing it for any other package. Also I like "jsnext:main" because it makes it more clear that it's a "beta" option ("module" is not yet standardized and thus I'm worried about potentially breaking the ecosystem once it is standardized).